tobac/merge_split.py

```diff
@@ -0,0 +1,231 @@
+"""
+    Tobac merge and split
+    This submodule is a post processing step to address tracked cells which merge/split. 
+    The first iteration of this module is to combine the cells which are merging but have received
+    a new cell id (and are considered a new cell) once merged. In general this submodule will label merged/split cells
+    with a TRACK number in addition to its CELL number.
+    
+"""
+
+
+def merge_split_cells(TRACK, dxy, distance=25000, frame_len=5):
```

Comment:
Should we require `distance` instead of making it optional or maybe make it dependent on `dxy` when the user does not provide it? I think this would make more sense given that this is really dependent on the data you are using and 25000 could be less than `dxy` in some datasets while it may be much higher than `dxy` in other datasets in which case `merge_split_cells()` becomes very expensive.

Reply:
If we make it dependent, a good educated guess would probably be the search radius defined via `v_max` or `d_max` in `linking_trackpy`, since this should already be finetuned with respect to the data.

Reply:
I thought of `dxy` since it already is an input parameter but your idea is a good one and makes actually more sense. I am not opposed to require `v_max` from the tracking, but if it feels too complicated we could simply make `distance` mandatory. @kelcyno do you have any opinion?

Reply:
I think we should make `distance` mandatory. @kelcyno are you ok with this?